enterprise/server/remote_execution/containers/bare/bare.go

```diff
@@ -44,7 +59,8 @@ func (p *Provider) New(ctx context.Context, _ *container.Init) (container.Comman
 type bareCommandContainer struct {
 	opts    *Opts
 	signal  chan syscall.Signal
-	WorkDir string
+	workDir string
+	tempDir string
```

Comment:
Nit: Having `EnableTmpdir` and `tempDir` as names is understandable but inconsistent. Would love to use either `tmp` or `temp` consistently.